### DISCUSSION TITLE
Clickhouse Backup options

### DISCUSSION THREAD

**Discussion by nomercy360** [Altinity/clickhouse-backup#754] [Q&A] [answered]
Hi, can you describe what are the differences between [Clickhouse built-in tools](https://clickhouse.com/docs/en/operations/backup) and `clickhouse-backup` tool. As i see they also have backup to S3 endpoint and S3 disk. What are use-cases and pros/cons of each approach?

**Comment by Slach** [comment, marked as answer]
`clickhouse-backup` have a bit more features, but allows to back up only *MergeTree tables by default
`clickhouse-backup` can use build-in BACKUP/RESTORE command

BACKUP/RESTORE + `system.backup_log` available only since 23.9+ clickhouse-server release
build-in solution still not allow delete created backup
look details about other unimplemented features https://github.com/ClickHouse/ClickHouse/issues/49306
